src/PerformanceTests/Tests/Categories/Base.cs

```diff
@@ -61,17 +65,19 @@ static void LaunchAndWait(Permutation permutation)
             var sessionIdArgument = string.Format(" --sessionId={0}", SessionId);
 
             var exe = new FileInfo(permutation.Exe);
+            var arguments = PermutationParser.ToArgs(permutation) + processIdArgument + sessionIdArgument;
```

Comment:
Can't you do something like
```
var permutationArguments = PermutationParser.ToArgs(permutation);
var arguments = string.Format("{0} {1} {2}", permutationArguments, processIdArgument, sessionIdArgument);
```
I'd had to look at it carefully to see what the intention was there. Because PermutationParser.ToArgs() feels like a list is being returned, not a single string. So that concatenation wasn't clear to me.